Automattic/wp-calypso
# Jetpack Manage: Make plans selectable on the Issue License page.

The purpose of this pull request is to allow selection of Plans on the Issue Licenses page.

**Before**
<img width="1557" alt="Screen Shot 2023-11-27 at 6 15 09 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/99e35f41-2bb3-4190-a7e2-4782baad6247">


**After**
<img width="1552" alt="Screen Shot 2023-11-27 at 6 14 18 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/e67ceb42-2337-4d7a-ac27-0bd66d308fc1">


Closes https://github.com/Automattic/jetpack-genesis/issues/119

## Proposed Changes

* Update the plan list to use the `LicenseProductCard` component instead of the `LicenseBundleCard` component.
* Add `withBackground` props to the LicenseProductCard component to support the visual cue for the Plans card.

## Testing Instructions

Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb. Make sure to switch it back to the previous type.

Use the Jetpack cloud live link below and go to the Licenses page (`/partner-portal/issue-license?flags=jetpack/bundle-licensing`)
* Confirm that the plans are now selectable.

## Pre-merge Checklist
